src/Symfony/Component/Console/Tests/Fixtures/DisabledCommand.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+use Symfony\Component\Console\Command\Command;
+
+class DisabledCommand extends Command
+{
+    protected function configure()
+    {
+        $this->setName('disabled');
+    }
+
+    public function isEnabled()
```

Comment:
This is a test fixture. We don't add PhpDoc there. But the `namespace` declaration is missing.